### PR TITLE
Never export samples/sample sets. Add dataset_name column to entities. Show export dialog even with no values selected.

### PR DESCRIPTION
**Pull request by wnojopra**
screenshot from 2019-02-28 12-25-17

### CODE REVIEW

**Comment by wnojopra** [comment]
Got rid of the checkbox. Now we just never export samples/sample sets.

Also added the dataset_id: 
screenshot from 2019-02-28 13-53-21

Also fixed the dialog when there is no facets selected:
https://screencast.googleplex.com/cast/NDY3MjAwMjE3ODY3ODc4NHxlNTEyNDA4OC1kZA

**Comment by melissachang** [comment]
Fixes #289

File: api/data_explorer/controllers/export_url_controller.py [hunk 1]
```diff
@@ -113,41 +113,22 @@ def _get_entities_dict(cohort_name, query, filter_arr):
...
+                'dataset_id': table_name.split('.')[1],
             }
         })
 
-    # If a cohort was selected, create a query entity and get Elasticsearch documents
-    # for the cohort so we can create sample_set entity.
+    # If a cohort was selected, create a query entity
     if cohort_name:
         entities.append({
             'entityType': 'cohort',
```
Comment (melissachang) [inline api/data_explorer/controllers/export_url_controller.py]: Right now there's no SQL query for the all_participants cohort. Let's do:
- Rename _get_filter_query -> _get_sql_query
- In _get_sql_query, if filter is empty, pick any participant table and return participant_ids from it
- Add TODO like

```
TODO: _get_sql_query() assumes that all tables have all participant ids.
Make _get_sql_query() work if that's not the case. participant ids should be
aggregated from all tables.
```

File: api/data_explorer/controllers/export_url_controller.py [hunk 2]
```diff
@@ -113,41 +113,22 @@ def _get_entities_dict(cohort_name, query, filter_arr):
...
             'attributes': {
-                'table_name': table_name
+                'table_name': table_name,
+                'dataset_id': table_name.split('.')[1],
             }
         })
 
-    # If a cohort was selected, create a query entity and get Elasticsearch documents
-    # for the cohort so we can create sample_set entity.
+    # If a cohort was selected, create a query entity
```
Comment (melissachang) [inline api/data_explorer/controllers/export_url_controller.py]: Delete 121-122

File: api/data_explorer/controllers/export_url_controller.py [hunk 3]
```diff
@@ -113,41 +113,22 @@ def _get_entities_dict(cohort_name, query, filter_arr):
             # underscores here and periods in table_name attribute.
```
[the remark this answers is not in the document]
Comment (wnojopra) [reply api/data_explorer/controllers/export_url_controller.py]: Can project ids have periods? "google.com:api-project-123"

And why are we splitting on the colon? Based on your description it sounds like we should be splitting on '.' instead.

I think what we actually want to do `'name': table_name.split('.')[-1]

**Comment by wnojopra** [comment]
@melissachang I made the updates, lmk what you think.

**Pull request merged**